apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/lib/getIncidentDescription.ts

```diff
@@ -46,7 +47,12 @@ export const getIncidentDescription = (
     policeCaseNumber,
   } = indictmentCount
 
-  const incidentLocation = crimeScene?.place || '[Vettvangur]'
+  const incidentLocation = crimeScene?.place
+    ? crimeScene.place
+        .split(',')
+        .map((str, idx) => applyCase(idx > 0 ? 'þgf' : 'þf', str))
```

Comment:
will this indexing pattern always apply to the entered places by the prosecutor? 👀

Reply:
No, it does not always applya. This is just best effort.

Commonly, we get the crime scene location as a street, followed by a comma and a town. This implementation will convert such places to the correct case, provided Beygla knows how to convert them (which is not always the case).

The implementation will also handle more complex places, such as street, town, province, as long as Beygla knows how to convert the individual components.

Anything Beygla does not know how to convert, it leaves unchanged, so that part of the place will not change.

My idea is to start with this simple approach and evolve it if needed.

Reply:
I will add comments to this code snippet.